Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.cs

```diff
@@ -104,8 +106,19 @@ private void CycleBlades()
             foreach (var blade in Items.OfType<BladeItem>())
             {
                 if (blade.IsOpen)
-                {
                     ActiveBlades.Add(blade);
+            }
+
+            // For now we skip this feature when blade mode is set to fullscreen
+            if (AutoCollapseCountThreshold > 0 && BladeMode != BladeMode.Fullscreen && ActiveBlades.Any())
+            {
+                var openBlades = ActiveBlades.Where(item => item.TitleBarVisibility == Visibility.Visible).ToList();
+                if (openBlades?.Count > AutoCollapseCountThreshold)
+                {
+                    for (int i = 0; i < openBlades.Count - 1; i++)
+                    {
+                        openBlades[i].BladeItemMode = BladeItemMode.Small;
```

Comment:
All open blades except the last one, IF the amount of blades drawn is over the threshold.